lib/theme_template/example/index.html

```diff
@@ -0,0 +1,14 @@
+---
+# Specify a layout from your theme!
+---
+
+Lorem ipsum dolor sit amet, quo id prima corrumpit pertinacia, id ius dolor dolores, an veri pertinax explicari mea. Agam solum et qui, his id ludus graeco adipiscing. Duis theophrastus nam in, at his vidisse atomorum. Tantas gloriatur scripserit ne eos. Est wisi tempor habemus at, ei graeco dissentiet eos. Ne usu aliquip sanctus conceptam, te vis ignota animal, modus latine contentiones ius te.
```

Comment:
@benbalter When a user creates a new theme using this command, they get a `bundle exec rake preview` command for free, which shows them this `example` website. Should we use this page to explain what to do, or just leave it to the README?